tests/unit/test_options_config.py

```diff
@@ -218,3 +218,15 @@ def test_parse_config_ignores_unknowns(tmp_path, opt_manager, caplog):
 def test_load_config_missing_file_raises_exception(capsys):
     with pytest.raises(exceptions.ExecutionError):
         config.load_config("foo.cfg", [])
+
+
+@pytest.mark.parametrize("option", ("ignore", "extend-ignore"))
+def test_invalid_ignore_codes_raise_error(tmpdir, opt_manager, option):
+    tmpdir.join("setup.cfg").write(
+        f"[flake8]\n{option} = E203, // this is invalid comment syntax"
+    )
+    with tmpdir.as_cwd():
+        cfg, _ = config.load_config("setup.cfg", [], isolated=False)
+
+    with pytest.raises(configparser.ParsingError):
```

Comment:
if you're raising a specific exception with a specific message that should be tested:

```python
with pytest.raises(...) as excinfo:
    ...
msg, = excinfo.value.args
assert msg == ...
```